generic/tests/vlan.py

```diff
@@ -91,11 +90,21 @@ def rem_vlan(test, session, v_id, iface="eth0", cmd_type="ip"):
                               test.log.info)
         return session.cmd_status(rem_vlan_cmd)
 
+    def find_free_port(session):
+        """
+        Returns a free port on the host
+        """
+        check_cmd = "netstat -nultp |awk '(NR>2){print $4}' | awk -F':' '{print $NF}'"
+        used_port_names = session.cmd_output(check_cmd).strip().split('\n')
```

Comment:
```suggestion
        used_port_names = session.cmd_output(check_cmd).strip().splitlines()
```

Reply:
Please help review it again, this line has been replaced, thanks a lot.